infrastructure/main.tf

```diff
@@ -100,6 +100,9 @@ module "bulk-scan" {
     SCAN_ENABLED                  = "${var.scan_enabled}"
     STORAGE_BLOB_LEASE_TIMEOUT    = "${var.blob_lease_timeout}" // In seconds
 
+    STORAGE_BLOB_SIGNATURE_ALGORITHM = "none" // none or sha256withrsa
+    STORAGE_BLOB_PUBLIC_KEY          = "${data.azurerm_key_vault_secret.exela-public-key.value}"
```

Comment:
should exela name be here?

Reply:
It would be actually better if it was clear whose public key that is. @njrich28 is this an issue?

Reply:
it was initially. rafal's question is valid

Reply:
true ... function rather than company name would be a better fit :)